doc/nrf/ug_bootloader_external_flash.rst

```diff
@@ -0,0 +1,67 @@
+.. _ug_bootloader_external_flash:
+
+Using external flash memory partitions
+######################################
+
+It is possible to use the external flash memory as the storage partition for the secondary slot.
+This requires a driver for the external flash memory that supports the following:
+
+* Single-byte read and write.
```

Comment:
all zephyr drivers must supports single-byte read
single byte write is not a requirement for mcuboot.
@hakonfam Did I miss something?

Reply:
@sigvartmh perhaps you can help clarify this?